htdocs/core/modules/user/doc/doc_generic_user_odt.modules.php

```diff
@@ -420,12 +420,12 @@ function write_file($object,$outputlangs,$srctemplatepath,$hidedetails=0,$hidede
 		return -1;
 	}
 
-	function get_substitutionarray_object($object,$outputlangs) {
+	function get_substitutionarray_object($object,$outputlangs,$array_key='object') {
 		foreach($object as $key => $value) {
-			if(!is_array($value) && !is_object($value)) {
-	    	$array_other['object_'.$key] = $value;
+			if (!is_array($value) && !is_object($value)) {
+				$array_other['object_'.$key] = $value;
```

Comment:
So here we should have 
$array_other[$array_key.'_'.$key] = $value;
?

Reply:
@eldy 
i don't know, i just add the missing parameter for avoid warning ! :-)

Reply:
@eldy fixed